src/main/java/org/jfrog/hudson/pipeline/action/DeployedMavenArtifact.java

```diff
@@ -1,31 +1,7 @@
 package org.jfrog.hudson.pipeline.action;
 
-import java.io.Serializable;
-
-public class DeployedMavenArtifact implements Serializable {
-    private static final long serialVersionUID = 1L;
-
-    private final String artifactoryUrl;
-    private final String repository;
-    private final String remotePath;
-    private final String name;
-
+public class DeployedMavenArtifact extends DeployedArtifact {
```

Comment:
It looks like the ```DeployedMavenArtifact``` and ```DeployedGradleArtifact``` classes are identical, and are therefore redundant. It is enough to have the ```DeployedArtifact``` class. There's a cxhance that this will lead to additional simplification is some of the APIs, and I'd be happy to review this PR again following the changes.

Reply:
The `DeployedMavenArtifact` class is present in the resulting xml files of the jobs. That means removing `DeployedMavenArtifacts` will break the result of previous jobs, showing an empty artifacts list. I could have added a `PassthruConverter` but that felt like quite an overkill for this simple view. Let me know what you think.

Reply:
I think that cost of leaving two redundant classes is higher than the cost of the converter. Before adding the converter, I suggest that you consult with @Or-Geva, who I believe has recently used Jenkins converters, to make sure they are implemented the right way. For example, we shouldn't care about downgrade compatibility in this specific case IMO.

Reply:
I removed the separate classes as we discussed